.github/workflows/test.yml

```diff
@@ -14,9 +18,6 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v1
-      - name: Run tests
-        run: |
+      - run: |
           npm install
           npm test
-        env:
```

Comment:
removed this as setting this to true, treats npm warnings as errors & fails the action.